Core/clim-core/table-formatting.lisp

```diff
@@ -67,37 +67,42 @@
    :align-x :left :align-y :baseline :min-width 0 :min-height 0))
 
 (defgeneric invoke-formatting-cell
-    (stream cont &key align-x align-y min-width min-height record-type &allow-other-keys)
-  (:method (stream continuation &key
-                                  (align-x :left)
-                                  (align-y :baseline)
-                                  (min-width 0)
-                                  (min-height 0)
-                                  (record-type 'standard-cell-output-record)
-                                  &allow-other-keys)
-    (invoke-with-new-output-record
-     stream
-     (lambda (stream record)
-       (declare (ignore record))
-       (with-temporary-margins (stream :left '(:absolute 0))
-         (letf (((stream-cursor-position stream) (values 0 0)))
-           (funcall continuation stream))))
-     record-type
-     :align-x align-x
-     :align-y align-y
-     :min-width (parse-space stream min-width :horizontal)
-     :min-height (parse-space stream min-height :vertical))))
+    (stream continuation
+     &key record-type align-x align-y min-width min-height &allow-other-keys)
+  (:method (stream continuation &rest initargs
+                                &key (record-type 'standard-cell-output-record)
+                                     (align-x :left)
+                                     (align-y :baseline)
+                                     (min-width 0)
+                                     (min-height 0)
+                                &allow-other-keys)
+    (apply #'invoke-with-new-output-record
+           stream
+           (lambda (stream record)
+             (declare (ignore record))
+             (with-temporary-margins (stream :left '(:absolute 0))
+               (letf (((stream-cursor-position stream) (values 0 0)))
+                 (funcall continuation stream))))
+           record-type
+           :align-x align-x
+           :align-y align-y
+           :min-width (parse-space stream min-width :horizontal)
+           :min-height (parse-space stream min-height :vertical)
+           (alexandria:remove-from-plist
+            initargs :record-type :align-x :align-y :min-width :min-height))))
```

Comment:
Instead of removing intiargs, this should be
```
(with-keywords-removed (initargs (:record-type))
  (apply stream lambda record-type initargs))
```

because otherwise when some other method verifies, whether the argument was supplied (for whatever reason), it will always conclude that yes, it was.

Reply:
See previous comments. But in this case, I don't see how the initargs could just be applied unmodified. That would forego the defaulting as well as the `parse-space` calls.

Reply:
ditto, closing